geometry_msgs/PoseStamped target_pose
float32 duration
bool precise_positioning
---
bool success
string message
---
string feedback